webui/vite.config.js

```diff
@@ -1,11 +1,18 @@
 import reactRefresh from '@vitejs/plugin-react-refresh';
 import eslintPlugin from 'vite-plugin-eslint';
-
+import replace from '@rollup/plugin-replace';
 
 // https://vitejs.dev/config/
 export default ({ command }) => {
   const baseConfig = {
     plugins: [
+      replace({
```

Comment:
Sorcery

Reply:
Had to find a quick way to embed this information inside our app.